analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/ComplexProjectFunctionalTest.java

```diff
@@ -98,11 +100,10 @@ public void ensureAlignmentFileCreated()
                         .filteredOn(t -> !getVersion(t).contains("redhat"))
                         .satisfies(l -> {
                             // make sure the two dynamic dependencies not aligned exist and have versions similar to what we expect
-
                             assertThat(l).filteredOn(t -> "HdrHistogram".equals(getArtifactId(t)))
-                                    .hasOnlyOneElementSatisfying(t -> assertThat(getVersion(t)).startsWith("2."));
+                                    .asString().contains("\", \"2.");
                             assertThat(l).filteredOn(t -> "commons-lang3".equals(getArtifactId(t)))
-                                    .hasOnlyOneElementSatisfying(t -> assertThat(getVersion(t)).startsWith("3."));
+                                    .asString().contains("\", \"3.");
```

Comment:
Shouldn't this be startsWith not contains as its changing the semantics of the test otherwise?

Reply:
The `.filteredOn` returns a `Tuple` which looks like `("artifactId", "version")'.  I see now, I lost the part that pulls out only the version.